### PR TITLE
Reportback spring cleaning

### LINKED ISSUE #3636: Remove old Prove It code
Once we go live:
- [x] Remove the `reportback_image` array code in `dosomething_campaign.theme.inc`
- [x] Delete the `field_image_reportback_gallery` field from the DoSomething campaign feature / db
- [x] Remove old Step 4 from template
- [x] Clean up and remove old Reportback module form code.
- [x] Remove JS logic to check for whether using new or old Reportback.
- [x] Remove feature flag for enabling cropping.
- [x] Remove feature flag for toggling to show the new Reportback.

### PR DESCRIPTION
**Pull request by weerd**
## Fixes #3636

Officially removes old Reportback code, and updates to just use the new Reportback v2.0 codez!

@aaronschachter 
@DoSomething/front-end

### CODE REVIEW

**Comment by aaronschachter** [comment]
This one is a beaut :+1: value added ship it immediately

File: lib/themes/dosomething/paraneue_dosomething/js/app.js [hunk 1]
```diff
@@ -21,7 +21,6 @@ define("app", function (require) {
 
   require("campaign/Carousel");
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/js/app.js:22]: You can remove Carousel too! :smiley:

**Comment by sbsmith86** [comment]
Looks good to me :dancers:

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_campaign.scss [hunk 2]
```diff
@@ -4,16 +4,15 @@
 
 
 // Page Specific Components
-.campaign--action .container--prove,
+.campaign--action .container--prove {
+  background: $yellow;
+}
+
 .campaign--sms .container--do {
   background: $purple asset-url("images/campaign-pattern-default.png") repeat center 63px;
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_campaign.scss]: Is this correct? Pretty sure this should be removed.

File: lib/themes/dosomething/paraneue_dosomething/js/app.js (hunk 1 above)
Comment (aaronschachter) [inline lib/themes/dosomething/paraneue_dosomething/js/app.js:22]: We might want to keep carousel for the Reportback Permalink and viewing multiple images on one permalink... cc @mikefantini

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_campaign.scss (hunk 2 above)
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_campaign.scss]: I wasn't sure cause that applies to SMS which the Reportback form didn't touch.

Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_campaign.scss]: Must just be getting overridden then: https://github.com/DoSomething/dosomething/blob/4e88027df55d9dbb38e3341b4e19f6899109634a/lib/themes/dosomething/paraneue_dosomething/images/campaign-pattern-default.png

Worth removing the rule, and also that image while you're at it!

Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_campaign.scss]: Hokay!

**Comment by DFurnes** [comment]
Ship it! :shipit:

**Pull request merged**